packages/manager/apps/ips/src/data/hooks/ip/useUpsertIpRipeInformation.ts

```diff
@@ -0,0 +1,28 @@
+import { useMutation } from '@tanstack/react-query';
+import { ApiError } from '@ovh-ux/manager-core-api';
+import { upsertIpRipeInformation } from '@/data/api/postorput/upsertIpRipeInformation';
+
+export type UseUpsertIpRipeInformationParams = {
+  ip: string;
+  description: string;
+  netname: string;
+  onSuccess: () => void;
+  onError: (error: ApiError) => void;
+};
+
+export const useUpsertIpRipeInformation = ({
+  ip,
+  description,
+  netname,
+  onSuccess,
+  onError,
+}: UseUpsertIpRipeInformationParams) => {
+  return useMutation({
+    mutationFn: () =>
+      upsertIpRipeInformation({ ip, description, netname }).then(
+        (response) => response.data,
+      ),
+    onSuccess,
```

Comment:
I think you can inline the useMutation directly where you need it, it just bloat the code for no added value since it is probably used once and there is no mutation key
React